Setup/SetExchAVExclusions/Set-ExchAVExclusions.ps1

```diff
@@ -105,15 +105,27 @@ if (-not $ListRecommendedExclusions) {
         exit
     }
 
-    $mpStatus = $null
-    $mpStatus = Get-MpComputerStatus -ErrorAction SilentlyContinue
-    if ($null -eq $mpStatus) {
-        Write-Error "We cannot get Microsoft Defender information"
+    $checkCmdLet = $null
+    $checkCmdLet = Get-Command Get-MpComputerStatus -ErrorAction SilentlyContinue
+    if ($null -eq $checkCmdLet) {
+        Write-Error "Get-MpComputerStatus cmdLet is not available"
+        Write-Error "This script only sets Exclusions on Microsoft Defender"
```

Comment:
We should change all the Write-Error calls to either Write-Warning or Write-Host -ForegroundColor Red. Write-Error is awkward because it produces multiple lines of output. This means writing one error is noisy and writing two is even worse:

![image](https://github.com/microsoft/CSS-Exchange/assets/4518572/7aa415e6-7864-45ce-8abb-185b3954c7b6)

We should only Write-Error if this is really a fatal error where we're going to need a call stack. line number, etc. For this output, we don't need that.